.github/workflows/bazel.yml

```
@@ -614,7 +617,9 @@ jobs:
         uses: jfrog/setup-jfrog-cli@d0a59b1cdaeeb16e65b5039fc92b8507337f1559 # pin@v3
 
       - name: Set dry run environment variable
-        if: ${{ github.event_name != 'push' || github.repository_owner != 'magma' || github.ref_name != 'master' }}
+        if: |
+          ${{ github.event_name != 'push' || github.repository_owner != 'magma' ||
+          ( github.ref_name != 'master' && startsWith(github.ref_name, 'v1.' ) == 'false' }} )
```

```suggestion
          ( github.ref_name != 'master' && ! startsWith(github.ref_name, 'v1.' ) }} )
```
https://docs.github.com/en/actions/learn-github-actions/expressions#operators

---

I didn't dare because there was no such comparison in the code :wink: 
But I tested it and it seems to work, thanks :slightly_smiling_face: